features/include/pcl/features/impl/brisk_2d.hpp

```diff
@@ -152,8 +152,7 @@ pcl::BRISK2DEstimation<PointInT, PointOutT, KeypointT, IntensityT>::generateKern
           // adapt the sizeList if necessary
           const unsigned int size = static_cast<unsigned int> (std::ceil (((scale_list_[scale] * radius_list[ring]) + pattern_iterator->sigma)) + 1);
 
-          if (size_list_[scale] < size)
-            size_list_[scale] = size;
+          size_list_[scale] = std::min(size_list_[scale], size);
```

Comment:
This line causes 1 failure. See [separate branch](https://github.com/kunaltyagi/pcl/tree/eigen2) without this line and passing tests. Ideas why this line causes the error?

Reply:
Link to failed CI: https://kunaltyagi.visualstudio.com/pcl/_build/results?buildId=503&view=results